addon.py

```
@@ -2320,6 +2427,722 @@ def import_generated_asset_hunyuan_ai(self, name: str , zip_file_url: str):
+            # Try to create a node
+            node = temp_tree.nodes.new(node_type_name)
+            node_type = getattr(bpy.types, node_type_name)
+
+            # Get socket information
+            inputs = self._collect_socket_info(node, is_input=True)
+            outputs = self._collect_socket_info(node, is_input=False)
+
+            # Get property information
+            properties = self._collect_property_info(node_type)
+
+            return True, inputs, outputs, properties
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)